nvsvocprez/view/templates/collection.html

```diff
@@ -68,14 +68,13 @@ <h2>{{ collection.prefLabel.value }}</h2>
         {% set conforms_to_profiles = collection.conforms_to.value.split(",") %}
         <div>
         {% for profile in conforms_to_profiles %}
-          {% for alt in alt_profiles_json %}
-            {% if profile in alt['url'] %}
+            {% if profile in alt_profiles %}
+              {% set alt = alt_profiles[profile] %}
```

Comment:
I'm not sure how jinja handles things, but will this be ok if profile isn't in alt_profiles?

Reply:
@matcor-bodc I've tested this with a hardcoded value and it works just like standard python it seems. So no exception, it just returns false.